pipeline/bicleaner/requirements/bicleaner-ai.in

```diff
@@ -1 +1,3 @@
-bicleaner-ai==2.0
+# cython_hunspell doesn't have wheels for Python 3.10
```

Comment:
This comment doesn't make as much sense as it did when we were pointing at a git revision. Perhaps something like: "Note: this package is installed manually before the rest of the requirements", with a pointer to the kind?